lib/api/pokedex_api_spec/handler/pokemon_handler.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:pokedex/api/pokedex_api_spec/handler/api_client.dart';
+import 'package:pokedex/api/pokedex_api_spec/model/pokemon_model.dart';
+
+class PokemonApi {
+  final ApiClient apiClient;
+
+  PokemonApi(this.apiClient);
+
+  Future<PokemonModel> getPokemonList() async {
+    final queryParams = <String, dynamic>{};
+
+    final baseUri = Uri.parse(apiClient.baseUrl);
+    final uri = baseUri.replace(queryParameters: queryParams, path: '${baseUri.path}/?limit=20');
```

Comment:
Create a static var here for the "20" then move the "/?limit=" param constants since you're gonna need this in pagination.